llama-index-integrations/llms/llama-index-llms-notdiamond/README.md

```diff
@@ -0,0 +1,5 @@
+# LlamaIndex Llms Integration: Not Diamond
+
+[Not Diamond](https://notdiamond.ai) offers an AI-powered model router that automatically determines which LLM is best-suited to respond to any query, improving LLM output quality by combining multiple LLMs into a **meta-model** that learns when to call each LLM.
+
+We offer support for cost and latency tradeoffs, customized router training, and real-time router personalization. Learn more via [our documentation](https://notdiamond.readme.io/).
```

Comment:
When possible, I try to avoid the use of the word "we" to refer to the company. Rather, I will use language like "Not Diamond offers support... Learn more via the documentation." The purpose of this is to frame Not Diamond as a technology that's "owned" by everyone, and avoid the you vs we distinction. Instead, I use "we" to describe our company _and_ the user, such as "Let's get started."